src/Connection/MessageReceiver.ts

```diff
@@ -367,6 +371,24 @@ export class MessageReceiver {
         }
     }
 
+    // Function: CheckForLicenseData
+    // Checks <licenseStateQueue> and <licenseChangeResponseQueue> for responses from the service
+    // related to Licensing. For those which match a callback in the relevant queue, invokes those
```

Comment:
🤷 This second lines feels weirdly worded

Reply:
`Invokes any matching callbacks from the relevant queue` sound clearer?

Reply:
👍 Sounds good to me